server/config/settings/development.py

```diff
@@ -0,0 +1,5 @@
+from .base import *  # noqa
+
+INSTALLED_APPS += (
+    "django_extensions",
+)
```

Comment:
If we are going to have time then we can try to make more universal base and use it in all out configs.